src/components/navigation-logo/navgation-logo.tsx

```diff
@@ -0,0 +1,74 @@
+import { Component, Element, h, Host, Prop, VNode } from "@stencil/core";
+import { CSS } from "./resources";
+
+@Component({
+  tag: "calcite-navigation-logo",
+  styleUrl: "navigation-logo.scss",
+  shadow: {
+    delegatesFocus: true
+  }
+})
+export class CalciteNavigationLogo {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+  @Element() el: HTMLCalciteNavigationLogoElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+  /** When true, the component is highlighted. */
+  @Prop({ reflect: true }) active: boolean;
+
+  /** Specifies the URL destination of the component, which can be set as an absolute or relative path.*/
+  @Prop({ reflect: true }) href: string;
+
+  /** Describes the appearance or function of the `thumbnail`. If no label is provided, context will not be provided to assistive technologies. */
+  @Prop() label: string;
+
+  /** Specifies the subtext to display, such as an organization or application description. */
+  @Prop() subtext: string;
```

Comment:
Should this maybe be named `description` to be consistent with other props?

Reply:
we can use description here considering we change the `text` attribute to something like `name` or `label`.

cc @jcfranco

Reply:
changed to heading and description. Wondering if we should drop `textEnabled` prop and render `heading` & `description` if provided by the user.